### PR TITLE
AVX2 implementation for memmove

### PR DESCRIPTION
**Pull request by vinaykompella**
Improves AnTuTu v9 "RAM Bandwidth" by 5%

Tracked-On: OAM-103726

### CODE REVIEW

File: aosp_diff/preliminary/bionic/0006-avx2-implementation-for-memmove-api.patch [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+
+diff --git a/libc/Android.bp b/libc/Android.bp
+index e4f40b323..5d7accf88 100644
+--- a/libc/Android.bp
++++ b/libc/Android.bp
+@@ -1019,6 +1019,7 @@ cc_library_static {
+                 "arch-x86_64/kabylake/string/avx2-memchr-kbl.S",
+                 "arch-x86_64/kabylake/string/avx2-memrchr-kbl.S",
+                 "arch-x86_64/kabylake/string/avx2-memset-kbl.S",
++                "arch-x86_64/kabylake/string/avx2-memmove-kbl.S",
```
Comment (shyjumon-n) [inline aosp_diff/preliminary/bionic/0006-avx2-implementation-for-memmove-api.patch:27]: Are we using the kabylake folder name ?

**Review by shyjumon-n — COMMENTED** [review]
Check comment and update

File: aosp_diff/preliminary/bionic/0006-avx2-implementation-for-memmove-api.patch (hunk 1 above)
Comment (vinaykompella) [reply aosp_diff/preliminary/bionic/0006-avx2-implementation-for-memmove-api.patch:27]: Traditionally it was used, but right now there is no such need for this. However, some configuration like cache.h still exists specific to architecture, hence a separate folder with that name. With DCC that also should go away.

**Review by shyjumon-n — APPROVED** [review]
LGTM

**Pull request merged**